snuba/datasets/tags_column_processor.py

```diff
@@ -176,7 +176,7 @@ def __extract_top_level_tag_conditions(self, condition: Expression) -> List[str]
                 isinstance(condition.parameters[0], FunctionCall)
                 and condition.parameters[0].function_name == "arrayJoin"
                 and condition.parameters[0].parameters
-                == (Column(None, "tags.key", None),)
+                == (Column(None, None, "tags.key"),)
```

Comment:
Before fixing this I need to actually implement the components that resolves tables and columns after parsing.
Then this will be:
```
Column(None, None, "tags", ("key",))
```